### PR TITLE
AppKit b3 changes

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: src/appkit.cs [hunk 1]
```diff
@@ -9391,6 +9391,10 @@ public enum NSImageName
 		[Mac (10, 12, 2)]
 		[Field ("NSImageNameTouchBarVolumeUpTemplate")]
 		TouchBarVolumeUpTemplate,
+
+		[Mac (10, 13)]
+		[Field ("NSImageNameTouchBarRemoveTemplate")]
+		TouchBarRemoveTemplate
```
Comment (rolfbjarne) [inline src/appkit.cs]: Minor: missing trailing comma.

File: src/appkit.cs [hunk 2]
```diff
@@ -11323,6 +11331,14 @@ partial interface NSPasteboard // NSPasteboard does _not_ implement NSPasteboard
 		[Field ("NSPasteboardTypeFindPanelSearchOptions")]
 		NSString NSPasteboardTypeFindPanelSearchOptions { get; }
 
+		[Mac (10, 13)]
+		[Field ("NSPasteboardTypeURL")]
+		NSString NSPasteboardTypeURL { get; }
```
Comment (rolfbjarne) [inline src/appkit.cs]: `Url`, not `URL`

File: src/appkit.cs [hunk 3]
```diff
@@ -11323,6 +11331,14 @@ partial interface NSPasteboard // NSPasteboard does _not_ implement NSPasteboard
...
+
+		[Mac (10, 13)]
+		[Field ("NSPasteboardTypeFileURL")]
+		NSString NSPasteboardTypeFileURL { get; }
```
Comment (rolfbjarne) [inline src/appkit.cs]: `Url`, not `URL`

Comment (chamons) [inline src/appkit.cs]: I really need to burn Url in my head, it always "looks wrong".

**Pull request merged**